installer/src/main/bin/validateModel.cmd

```diff
@@ -65,7 +74,6 @@ if "%SHOW_USAGE%" == "false" (
 ECHO.
 ECHO Usage: %SCRIPT_NAME% [-help]
 ECHO              -oracle_home ^<oracle_home^>
-ECHO              [-print_usage ^<context^> [-attributes_only^|-folders_only^|-recursive] ]
 ECHO              [-model_file ^<model_file^>]
```

Comment:
model file is now required for validate (print usage you did not give model file)

Reply:
Model can be in the archive file